include/lsst/jointcal/CcdImage.h

```diff
@@ -28,238 +29,238 @@ typedef int ShootIdType;
-  unsigned BandRank() const {return bandRank;}
+    //! instrument (TOADINST fits pseudo-key)
+    std::string Instrument() const {return instrument;}
 
-  //! returns seeing
-  //  double Seeing() const { return seeing;}
+    //! some incremental band rank. Is used to incrementally index bands in a sample of input images. Different from BandIndex()
+    unsigned BandRank() const {return bandRank;}
 
-  //! returns gfseeing
-  //  double GFSeeing() const { return gfseeing;}
+    //! returns seeing
+    //  double Seeing() const { return seeing;}
 
-  //! returns sigma back
-  //  double SigmaBack() const { return sigmaback;}
+    //! returns gfseeing
+    //  double GFSeeing() const { return gfseeing;}
 
-  //! returns shoot ID
-  ShootIdType Shoot() const { return shoot;}
+    //! returns sigma back
+    //  double SigmaBack() const { return sigmaback;}
 
-  //! Exposure time (s)
-  double ExpTime() const { return expTime;}
+    //! returns shoot ID
+    ShootIdType Shoot() const { return shoot;}
 
-  //!  Airmass
-  double AirMass() const {return airMass;}
+    //!  Airmass
+    double AirMass() const {return airMass;}
 
-  //! Date Obs
-  std::string DateObs() const { return dateObs; }
+    //! Date Obs
+    std::string DateObs() const { return dateObs; }
 
-  //! Julian Date
-  double JD() const { return jd; }
+    //! Julian Date
+    double getMjd() const { return mjd; }
 
 
-  //!Elixir ZP (applies to fluxes in ADU/sec at airmass 1).
-  double ElixirZP() const { return elixirZP;}
+    //!Elixir ZP (applies to fluxes in ADU/sec at airmass 1).
+    double ElixirZP() const { return elixirZP;}
 
-  //!zp from the Fits key set by SetZpKey(std::string)
-  double ZP() const { return zp;}
+    //!zp from the Fits key set by SetZpKey(std::string)
+    double ZP() const { return zp;}
 
-  //!zp from the psf zp file, returns 0 if not present
-  double PSFZP() const { return psfzp;}
+    //!zp from the psf zp file, returns 0 if not present
+    double PSFZP() const { return psfzp;}
 
-  //! absorption term
-  double PhotK() const { return photk; }
+    //! absorption term
+    double PhotK() const { return photk; }
 
-  //! original ZP
-  double PhotC() const { return photc; }
+    //! original ZP
+    double PhotC() const { return photc; }
 
-  //!
-  double HourAngle() const { return hourAngle; }
+    //!
+    double HourAngle() const { return hourAngle; }
 
-  //! Parallactic angle
-  double SinEta() const { return sineta; }
+    //! Parallactic angle
+    double SinEta() const { return sineta; }
 
-  //! Parallactic angle
-  double CosEta() const { return coseta; }
+    //! Parallactic angle
+    double CosEta() const { return coseta; }
 
-  //! Parallactic angle
-  double TanZ() const { return tgz; }
+    //! Parallactic angle
+    double TanZ() const { return tgz; }
 
-  //!
-  Point ParallacticVector() const {return Point(tgz*coseta, tgz*sineta);}
+    //!
+    Point RefractionVector() const {return Point(tgz * coseta, tgz * sineta);}
```

Comment:
Oof. I've updated my SublimeAStyleFormatter config (that's what tdid the cleanup) to not put spaces around operators. I'll see if I can easily get these sorted out manually.